enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,381 @@
+an 120 Gb boot image. This is too long to meet our target of being able to install a whole cloud within the hour. One of
+the reasons this is slow is because we cannot do the image resizing after the copy, since it's writing directly to a
+sized disk. Once imported, creating the nodes is fast, since the other disks are flashed from the snapshot. For now, we
+asked the team to look into meeting the requirement that importing the image can take 5 minutes or less. Another bit
+we're working on is defining the implementation details for how the image will be imported from the COS bucket to the
+PowerVS service. Currently, the terraform API doesn't support importing an image from bucket. We are looking into if it
+is possible to update the terraform provider to do this with support for optional keys to support both private and
+public buckets. IBM Cloud Object Storage supports public buckets, but the terraform provider API will need an update to
+make keys optional in order to support this workflow.
+
+This will happen as part of the terraform provider.
+
+~~2. Published CoreOS Images to the IBM Cloud Pre-Authorized Default Images This approach would involve outputting our
+CoreOS OVA images in a queue where they can be pre-loaded into the IBM Cloud so that they are available as soon as the
+Virtual Server instance is available. This is significantly quicker than the method above but has several drawbacks. For
+one, these default images need to be pre-authorized in order for them to be added to the IBM Cloud. This is currently a
+manual process owned by the Power VS team, so it cannot be completed by a user. Additionally, while we could build
+automation to solve this, it's unclear how would allow the user to point to the correct CoreOS image in the default
+list, since they would have to know the image name and details before their instance is created. Finally, we don't want
+to muddy up the default image list with an ever growing list of CoreOS options. This pruning would need to be automated
+some other mechanism would need to be in place to protect the user experience.~~
+
+### Open Questions
+
+#### OVA Import Time
+Is 5 minutes for important a target deployment image satisfactory for an IPI deployment. Additonally, is it even
+possible for the Power VS team to reduce the current import time from 30 minutes to 5 (for the purposes of meeting this
+requirement)?
+
+#### Deployment Time
+What is the maximum allowable time for a successful deployment to take. Is under an hour sufficient?
+
+#### DHCP in the IBM Cloud
+The commitment we were given by the IBM Cloud team for this requirement is October 18th, 2021. As of early November,
+this was delivered in a staging environment, tested, and will be going live to a production system in the coming weeks.
+
+At this time, we don't have any alternatives since this is a hard requirement for IPI.
```

Comment:
This may need an update now?

Reply:
We've had DHCP for a while now, so I've removed references to it being an open question.